src/lqcut.jl

```diff
@@ -168,8 +159,10 @@ function LQ_cut(
 
     #get truncate values for fit; needed if outliers are present after in sideband subtracted histogram
     lq_prestats = estimate_single_peak_stats(hist_corrected)
-    lq_start = lq_prestats.peak_pos - truncation_sigma * lq_prestats.peak_sigma
-    lq_stop = lq_prestats.peak_pos + truncation_sigma * lq_prestats.peak_sigma
+    lq_start = lq_prestats.peak_pos - cut_truncation_sigma * lq_prestats.peak_sigma
+    lq_stop = lq_prestats.peak_pos + cut_truncation_sigma * lq_prestats.peak_sigma
+
+    #simplecuts benutzen
 
     # Fit the sideband subtracted histogram
     fit_result, fit_report = fit_binned_trunc_gauss(hist_corrected, (low=lq_start, high=lq_stop, max=NaN))
```

Comment:
I think, uncertainties here would be nice.

Reply:
I would also add more stuff into the result, in particular about this fit. Since otherwise, you cannot debug anything later.

Reply:
The `fit_binned_trunc_gauss` returns values with uncertainties or do you mean something else?

Reply:
Yes, but set the `uncertainty=true` kwarg explicitly.

Reply:
The fit is saved in the report. In gerneral what should be inside the result next to final values, like the cut, or the property function to get the classifier?

Reply:
Whatever you put into the result, will be later saved on disk and can be accessed for each run. This is then the only chance to do monitoring etc. So you should think about what you want to plot later and then fill the `result` accordingly.